sdc/tests/test_dataframe.py

```diff
@@ -1236,6 +1236,17 @@ def test_impl(df, start, end):
         ref_result = test_impl(df, start, end)
         pd.testing.assert_frame_equal(jit_result, ref_result)
 
+    @skip_sdc_jit('DF.getitem unsupported Series name')
+    def _test_df_getitem_tuple_idx(self, df):
+        def test_impl(df):
+            # pd.df.getitem does not support idx as a tuple
+            return df[['A', 'C']]
+
+        # SDC pd.df.getitem does not support idx as a list
+        sdc_func = self.jit(lambda df: df[('A', 'C')])
```

Comment:
I think we should state this as a limitation in our documentation too.
Also it might look better if you write it this way:
```
        test_impl = lambda df: df[['A', 'C']]
        sdc_func = self.jit(lambda df: df[('A', 'C')])
```
or use do_jit parameter, e.g.
 ```
       # Pandas implementation does not support idx as a tuple
        # but SDC implementation only supports tuple, so adjust tested code
        def test_impl(df, do_jit):
            if do_jit == True:  # noqa
                return df[('A', 'C')]
            else:
                return df[['A', 'C']]
        sdc_func = self.jit(test_impl)

        pd.testing.assert_frame_equal(sdc_func(df, True), test_impl(df, False))
```